lib/gssapi/mech/gss_accept_sec_context.c

```diff
@@ -264,7 +264,9 @@ gss_accept_sec_context(OM_uint32 *minor_status,
 	    *delegated_cred_handle = GSS_C_NO_CREDENTIAL;
 	_mg_buffer_zero(output_token);
 
+        _gss_mg_log(10, "gss-asc: enter");
         if (!*context_handle) {
+                _gss_mg_log(10, "gss-asc: allocating new context");
```

Comment:
Do we want to log the pointer value for the context handle so as to be able to stitch together related calls?